### PR TITLE
Add irc-client.0.6.2

### PR DESCRIPTION
**Pull request by johnelse**

### CODE REVIEW

**Comment by c-cube** [comment]
TLS seems broken on 4.02 (lwt.syntax?? wow). Not really our fault there, the bound should probably be on tls.

File: packages/irc-client-lwt-ssl/irc-client-lwt-ssl.0.6.2/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/irc-client-lwt-ssl/irc-client-lwt-ssl.0.6.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
Dune shouldn't be tagged as a build dependency anymore (see #14266). Furthermore the `dune-project` files says this package requires at least dune 1.6

File: packages/irc-client-lwt/irc-client-lwt.0.6.2/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/irc-client-lwt/irc-client-lwt.0.6.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
Same here

File: packages/irc-client-tls/irc-client-tls.0.6.2/opam [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/irc-client-tls/irc-client-tls.0.6.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
Same here.

File: packages/irc-client-unix/irc-client-unix.0.6.2/opam [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/irc-client-unix/irc-client-unix.0.6.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
Same here

File: packages/irc-client/irc-client.0.6.2/opam [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/irc-client/irc-client.0.6.2/opam]:
```suggestion
  "dune" {>= "1.6"}
```
Same here

**Comment by kit-ty-kate** [comment]
> TLS seems broken on 4.02 (lwt.syntax?? wow). Not really our fault there, the bound should probably be on tls.

This is now fixed

**Comment by kit-ty-kate** [comment]
Thanks!

**Comment by johnelse** [comment]
Thanks!

**Pull request merged**